### PR TITLE
Localize “Good practices for Kubernetes Secrets” to Japanese

### PR DESCRIPTION
**Pull request by bezalel-7**
Hello!

issue: #42450 
This pull request addresses issue #42450. It adds the translated content of the `secrets-good-practices.md` document from English to Japanese.

The purpose of this change is to make the Kubernetes documentation more accessible to Japanese-speaking users by providing a translated version of the "Kubernetes Secretsの適切な使用方法" document.

Thank you for reviewing this pull request!

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Localize “Good practices for Kubernetes Secrets” to Japanese

File: content/ja/docs/concepts/security/secrets-good-practices.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Secretへの最小特権アクセスを構成する {#least-privilege-secrets}
+
+Kubernetesの{{<glossary_tooltip term_id="rbac" text="ロールベースアクセス制御">}} [(RBAC)](/docs/reference/access-authn-authz/rbac/)などのアクセス制御メカニズムを計画する際、`Secret`オブジェクトへのアクセスに関する以下のガイドラインを考慮してください。
+また、[RBACの適切な使用方法](/docs/concepts/security/rbac-good-practices)の他のガイドラインにも従ってください。
+
+- **コンポーネント**: `watch`または`list`アクセスを、特権のあるシステムレベルのコンポーネントに制限してください。コンポーネントの通常の動作が必要とする場合にのみ、Secretへの`get`アクセスを許可してください。
+- **ユーザー**: Secretsへの`get`、`watch`、`list`アクセスを制限してください。`etcd`へのアクセスはクラスタ管理者にのみ許可し、読み取り専用アクセスも許可してください。特定の注釈を持つSecretへのアクセスを制限するなど、より複雑なアクセス制御については、サードパーティの認証メカニズムを検討してください。
+
+{{< caution >}}
+Secretsへの`list`アクセスを許可すると、サブジェクトがSecretの内容を取得できるようになります。
```
Comment (sftim) [inline content/ja/docs/concepts/security/secrets-good-practices.md:40]:
```suggestion
Secretへの`list`アクセスを許可すると、サブジェクトがSecretの内容を取得できるようになります。
```

File: content/ja/docs/concepts/security/secrets-good-practices.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+title: Kubernetes Secretsの適切な使用方法
```
Comment (sftim) [inline content/ja/docs/concepts/security/secrets-good-practices.md:2]:
```suggestion
title: Kubernetes Secretの適切な使用方法
```

**Review by sftim — COMMENTED** [review]
I usually avoid using English-style plurals when localizing to languages with obviously different grammar.

**Comment by sftim** [comment]
Thanks for the PR @bezalel-7 !

File: content/ja/docs/concepts/security/secrets-good-practices.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+### Secretへの最小特権アクセスを構成する {#least-privilege-secrets}
+
+Kubernetesの{{<glossary_tooltip term_id="rbac" text="ロールベースアクセス制御">}} [(RBAC)](/docs/reference/access-authn-authz/rbac/)などのアクセス制御メカニズムを計画する際、`Secret`オブジェクトへのアクセスに関する以下のガイドラインを考慮してください。
+また、[RBACの適切な使用方法](/docs/concepts/security/rbac-good-practices)の他のガイドラインにも従ってください。
+
+- **コンポーネント**: `watch`または`list`アクセスを、特権のあるシステムレベルのコンポーネントに制限してください。コンポーネントの通常の動作が必要とする場合にのみ、Secretへの`get`アクセスを許可してください。
+- **ユーザー**: Secretsへの`get`、`watch`、`list`アクセスを制限してください。`etcd`へのアクセスはクラスタ管理者にのみ許可し、読み取り専用アクセスも許可してください。特定の注釈を持つSecretへのアクセスを制限するなど、より複雑なアクセス制御については、サードパーティの認証メカニズムを検討してください。
+
+{{< caution >}}
```
Comment (Sajiyah-Salat) [inline content/ja/docs/concepts/security/secrets-good-practices.md:39]: {{<caution>}}
{{</caution>}}
I think should be like this

**Review by Sajiyah-Salat — CHANGES_REQUESTED** [review]

File: content/ja/docs/concepts/security/secrets-good-practices.md (hunk 3 above)
Comment (t-inu) [reply content/ja/docs/concepts/security/secrets-good-practices.md:39]: According to the [Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/#caution), the following format is correct.
```
{{< caution >}}
{{< /caution >}}
```
On the other hand, the last sentnce in this PR uses `{{<caution>}} ... {{</caution>}}`.
This is the same notation as the original page, and works in this format as well.
https://github.com/kubernetes/website/blob/4d8feb6765a52362db09ef8d5dce41f45bc5fe9b/content/en/docs/concepts/security/secrets-good-practices.md?plain=1#L113-L116
If fix it to the correct notation, it is better to update the original page at first.

Comment (t-inu) [reply content/ja/docs/concepts/security/secrets-good-practices.md]: My comment means the correct format is **with spaces**.
And for multiple language modifications, need to create different PRs.

**Comment by bezalel-7** [comment]
Hi @t-inu,

I wanted to let you know that the changes made in the en docs (https://github.com/kubernetes/website/pull/42650) have been merged. Thank you for your contribution!

I've also applied the same changes to the "bezalel-7:bezalel-ja-docs" repository. Could you please review it when you have a moment and let me know if there's anything specific that should be changed or if everything looks good to you?

Your feedback is greatly appreciated.

Thanks again,
@bezalel-7

**Comment by bezalel-7** [comment]
> @bezalel-7
> 
> > I wanted to let you know that the changes made in the en docs (#42650) have been merged.
> 
> Great!
> 
> I'm reviewing now. It seems the change to `content/en/docs/concepts/security/secrets-good-practices.md` is still detected in this PR. The differece is the presence or absence of the last blank line. Could you please make the file in your repository have the same content as `main`?

done! @t-inu

**Comment by sftim** [comment]
@bezalel-7 please squash this to one commit, so that none of the commits in this PR make changes outside of the Japanese localization teams purview.

**Comment by bezalel-7** [comment]
> @bezalel-7 please squash this to one commit, so that none of the commits in this PR make changes outside of the Japanese localization teams purview.

hey @sftim i am doing it for the first time can you please tell me how to do it?

**Comment by atoato88** [comment]
It need to be remove the file which is not related with `ja`, because that the target contents in this PR is `ja`.
So could you remove the file "content/ja/docs/concepts/security/secrets-good-practices.md" in this PR?

/remove-language en
I removed the `language/en` label, because of same reason.

**Comment by bezalel-7** [comment]
> It need to be remove the file which is not related with `ja`, because that the target contents in this PR is `ja`. So could you remove the file "content/ja/docs/concepts/security/secrets-good-practices.md" in this PR?
> 
> /remove-language en I removed the `language/en` label, because of same reason.

 @atoato88 i will close this pr because i merge different contents so this is messed up ! https://github.com/kubernetes/website/pull/42730 i opened another pr with only ja changes done! you the reviewer of that ! can you please check that out

**Pull request closed without merging**